guides/09_Amazon_Voice_Focus.md

```diff
@@ -213,13 +260,76 @@ As noted above, the device controller must support Web Audio.
 
 Both `isSupported` and `create` accept _specifications_ — `VoiceFocusSpec` structures — as input. These allow you to describe the model attributes, execution approach, and other configuration options that define how the feature should operate.
 
+To use Amazon Voice Focus, the name section of the specification should be set to `'default'` whereas to use Echo Reduction, the name section of the specification should be set to `'ns_es'`. Echo Reduction can only be enabled while creating the meeting.
+
+```typescript
+spec = { ...
+   name: 'default' // for Amazon Voice Focus
+   ...
+}
+spec = { ...
+   name: 'ns_es' // for Echo Reduction 
```

Comment:
ns_es is for both right VF and Echo Reduction?

Reply:
yes

Reply:
can we update it to mention that. It seems we refer to `VF and Echo Reduction` everywhere but not here.